### ISSUE TITLE
508-defect-3: Headings should be used to start sections in a logical order

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#42233]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Details

The current heading order suggests that:
- the introductory "find" link begins a section (which it doesn't)
- VSO help is specific to what they need to upload

<img width="693" alt="Screen Shot 2022-06-02 at 2 52 43 PM" src="https://user-images.githubusercontent.com/14154792/171707906-77381632-f6f9-4c13-bb4c-f2651bd305aa.png">

## Acceptance Criteria

- [ ] The "find" link is not a heading (consider an action link)
- [ ] "What if I need" `strong` element is a `h4`
- [ ] Design reviews changes and validates from a Ux perspective.

## Proposed Solution (if known)
<img width="752" alt="Screen Shot 2022-06-02 at 2 50 09 PM" src="https://user-images.githubusercontent.com/14154792/171706268-11da88e4-c5de-4b34-a595-5dda3b5a94b0.png">

**Comment by joshkimux** [comment]
<img width="1298" alt="Screen Shot 2022-06-30 at 8 20 00 AM" src="https://user-images.githubusercontent.com/14154792/176675559-5163993b-45e3-41cb-aa6b-00e4166d3adb.png">

Brilliant-- thanks for the quick fix here @jerekshoe 💯
